modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -218,7 +273,7 @@ const Actions = ({fdsView, namespace, spritemap}: IFDSViewSectionProps) => {
 		loadFDSActions();
 
 		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, []);
+	}, [activeTab]);
```

Comment:
This is a bit of a red flag. Dependencies should be either fully covered, or there should be none (do this only on first render). Partial dependencies are a bad pattern, as they lead to stale states. Instead of change on this line, what we can do in directly invoke `loadFDSActions()` on `onActiveChange`.